src/Compilers/Core/Portable/EncodedStringText.cs

```diff
@@ -21,15 +21,15 @@ internal static class EncodedStringText
         /// </summary>
         private static readonly Encoding s_utf8Encoding = new UTF8Encoding(encoderShouldEmitUTF8Identifier: false, throwOnInvalidBytes: true);
 
+        private static readonly Lazy<Encoding> s_fallbackEncoding = new Lazy<Encoding>(GetFallbackEncoding);
+
         /// <summary>
         /// Encoding to use when UTF-8 fails. We try to find the following, in order, if available:
         ///     1. The default ANSI codepage
         ///     2. CodePage 1252.
         ///     3. Latin1.
         /// </summary>
-        private static readonly Lazy<Encoding> s_fallbackEncoding = new Lazy<Encoding>(GetFallbackEncoding);
-
-        private static Encoding GetFallbackEncoding()
+        internal static Encoding GetFallbackEncoding()
```

Comment:
```suggestion
        internal static Encoding CreateFallbackEncoding()
```

At the moment this has the same name as the method on `Compilation` but has very different implementation. The `Create` prefix here is more indicative of the implementation

Reply:
sgtm